src/main/java/io/github/railroad/project/ui/project/ProjectListCell.java

```diff
@@ -7,10 +7,12 @@
 import javafx.geometry.Pos;
 import javafx.scene.control.Label;
 import javafx.scene.control.ListCell;
+import javafx.scene.control.Separator;
 import javafx.scene.image.ImageView;
-import javafx.scene.layout.HBox;
+import javafx.scene.layout.*;
```

Comment:
Bit weird to do this, usually you only do that when there is `n`(usually about 5) or more items imported from that package.